tests/entrypoints/openai/basic_tests/test_chat_logit_bias_validation.py

```diff
@@ -7,9 +7,7 @@
 
 from vllm.config import ModelConfig
 
-from ...utils import RemoteOpenAIServer
-
-MODEL_NAME = "Qwen/Qwen2.5-1.5B-Instruct"
+MODEL_NAME = "hmellor/tiny-random-LlamaForCausalLM"
```

Comment:
Could this also be made a fixture so that we don't have to keep anything in sync with conftest?